helm/templates/metric-anomaly-detector/_helpers.tpl

```diff
@@ -0,0 +1,7 @@
+{{- define "metricanomalydetector.image" -}}
```

Comment:
could probably share this if you wanted to, same logic as earlier.

Reply:
done